src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/ComparisonExpression.java

```diff
@@ -33,8 +33,12 @@ public List<FilterOperation<?>> getFilterOperations() {
     return filterOperations;
   }
 
-  /** This method will be called when not operation is pushed down */
-  public void flip() {
+  /**
+   * implements Invertible, method to invert a ComparisonExpression this method will be called when
+   * $not operator is pushed down
+   */
+  @Override
+  public Object invert() {
     List<FilterOperation<?>> filterOperations = new ArrayList<>(this.filterOperations.size());
     for (FilterOperation<?> filterOperation : this.filterOperations) {
       final FilterOperator flippedOperator = filterOperation.operator().flip();
```

Comment:
Should we rename these variables to say invertedOpetator, since we are moving away from using `flip`? Also rename the Operator method (`operator().flip()`) to `invert`

Reply:
fixed.